### PR TITLE
fix: space wordmark out a bit

### PR DESCRIPTION
**Pull request by redreceipt**
before

Screen Shot 2022-06-14 at 2 22 30 PM

after

Screen Shot 2022-06-14 at 2 19 12 PM

### CODE REVIEW

File: packages/apollos-ui-onboarding/src/LandingSwiper/slides.js [hunk 1]
```diff
@@ -56,6 +48,16 @@ const ItemText = styled(({ theme }) => ({
...
+const SlideText = ({ title, subtitle }) => {
+  const theme = useTheme();
+  return (
+    <View style={styles.textContainer(theme)}>
+      <Title>{title}</Title>
+      <SubTitle secondary>{subtitle}</SubTitle>
+    </View>
+  );
+};
+
```
Comment (nlewis84) [inline packages/apollos-ui-onboarding/src/LandingSwiper/slides.js:60]: I like this.

**Review by nlewis84 — APPROVED** [review]

**Comment by redreceipt** [comment]
> Does this look good in Android?

should be the exact same. the only real change is margin bottom to the wordmark

**Comment by nlewis84** [comment]
I decided to just check on Android. I think we need to go back to forcing the line break via padding or something, or fix this flashing that is going on. @redreceipt check it out:

| Before | After |
|---|---|
| https://user-images.githubusercontent.com/72768221/173668886-5e36bc27-a288-4fc2-b0ab-a0b76742e9c7.mp4 | https://user-images.githubusercontent.com/72768221/173668953-2524a6f1-39d4-4d18-a38f-03d0be6ff7bd.mp4 |

**Comment by redreceipt** [comment]
oh shoot, i took out `width: 58%` :(

**Pull request merged**